src/validate.c

```diff
@@ -39,6 +39,20 @@ static bool validate_string_length( const char* str, size_t max_length ) {
   return validation_status;
 }
 
+bool validate_procid_length(const char* procid ) {
+  size_t length = strlen( procid );
+  bool validation_status = true;
+
+  if( length > 128 ) {
```

Comment:
use the `STUMPLESS_MAX_PROCID_LENGTH` value here as well instead of a raw integer.